protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing packets. " ..
```

Comment:
For outgoing encrypted UDP packets, not for the inner unencrypted packets. You might want to make this clearer in the wording.